build_scripts/windows/product.wxs

```diff
@@ -97,6 +109,32 @@
                      System="yes" />
                 <CreateFolder />
             </Component>
+            <Component Id="PromptflowCliRegistry"
+                       Directory="DynamicCliDir"
+                       Guid="$(var.PromptflowCliRegistryGuid)">
+                <RegistryValue Root="HKCU"
+                               Key="Software\Microsoft\$(var.ProductName)"
+                               Name="installed"
+                               Type="integer"
+                               Value="1"
+                               KeyPath="yes"/>
+            </Component>
+            <Component Id="PromptflowCliVersion"
+                       Directory="DynamicCliDir"
+                       Guid="$(var.PromptflowCliVersionGuid)">
+                <RegistryValue Root="HKLM"
+                               Key="Software\Microsoft\$(var.ProductName)"
+                               Name="version"
+                               Type="string"
+                               Value="$(var.ProductVersion)"
+                               KeyPath="yes"/>
+            </Component>
+            <Component Id="PromptflowService" Directory="DynamicCliDir" Guid="$(var.PromptflowServiceGuid)">
+                <File Id="PromptflowServiceEXE" Source="scripts\dist\promptflow\promptflow_service.exe" KeyPath="yes" />
+                <ServiceInstall Id="ServiceInstaller" Name="PromptflowService" DisplayName="Promptflow Service" Description="This is a promptflow service" Start="auto" Type="ownProcess" ErrorControl="normal" />
```

Comment:
promptflow -> prompt flow

Reply:
DisplayName: Prompt Flow Service
Description:  Provides run and connection management for prompt flow